examples/post.v3.LR.historical.zppy_v3.cfg

```diff
@@ -0,0 +1,289 @@
+
+  [[ atm_daily_180x360_aave ]]
+  # This subtask is a dependency for the e3sm_diags task's atm_monthly_180x360 subtask.
+  # The following e3sm_diags sets require it:
+  # "tropical_subseasonal"
+  frequency = "daily"
+  input_files = "eam.h1"
+  input_subdir = "archive/atm/hist"
+  mapping_file = /home/ac.zender/data/maps/map_ne30pg2_to_cmip6_180x360_aave.20200201.nc
+  # Needed for Wheeler Kiladis
+  vars = "FLUT,PRECT,U850"
+
+  [[ rof_monthly ]]
+  # The following e3sm_diags sets require it:
+  # "streamflow"
+  extra_vars = 'areatotal2'
+  frequency = "monthly"
+  input_files = "mosart.h0"
+  input_subdir = "archive/rof/hist"
+  mapping_file = ""
+  vars = "RIVER_DISCHARGE_OVER_LAND_LIQ"
+
+  [[ atm_monthly_glb ]]
+  # This subtask is a dependency for the global_time_series task.
+  frequency = "monthly"
+  input_files = "eam.h0"
+  input_subdir = "archive/atm/hist"
+  mapping_file = "glb"
+
+  [[ lnd_monthly_glb ]]
+  # This subtask is a dependency for the global_time_series task.
+  frequency = "monthly"
+  input_files = "elm.h0"
+  input_subdir = "archive/lnd/hist"
+  mapping_file = "glb"
+  # vars = "FSH,RH2M,LAISHA,LAISUN,QINTR,QOVER,QRUNOFF,QSOIL,QVEGE,QVEGT,SOILWATER_10CM,TSA,H2OSNO,TOTLITC,CWDC,SOIL1C,SOIL2C,SOIL3C,SOIL4C,WOOD_HARVESTC,TOTVEGC,NBP,GPP,AR,HR"
+  vars = "" # This will tell zppy to use all available variables
+
+[e3sm_to_cmip]
+# This is a new task in zppy v3.0.0!
+# This task replaces the `ts_fmt = "cmip"` functionality previously found in the `ts` task.
+active = True
+frequency = "monthly"
+# An alternative to setting `ts_grid` here is setting `ts_atm_grid` and `ts_land_grid` in the respective subtasks below.
+ts_grid = "180x360_aave"
+ts_num_years = 30
+walltime = "00:50:00"
+years = "1985:2014:30"
+
+  [[ atm_monthly_180x360_aave ]]
+  # This subtask is a dependency for the ilamb task.
+  # This subtask depends on the ts task's atm_monthly_180x360_aave subtask.
+  # Notice this subtask name matches a subtask in the `ts` task.
+  # If it did not, then the `ts_atm_subsection` parameter would be required here to tell zppy which subtask to use.
+  input_files = "eam.h0"
+
+  [[ land_monthly ]]
+  # This subtask is a dependency for the ilamb task.
+  # This subtask depends on the ts task's land_monthly subtask.
+  # Notice this subtask name matches a subtask in the `ts` task.
+  # If it did not, then the `ts_land_subsection` parameter would be required here to tell zppy which subtask to use.
+  input_files = "elm.h0"
+
+[tc_analysis]
+# This subtask is a dependency for the e3sm_diags task's atm_monthly_180x360_aave subtask.
+# 1 DEPRECATED PARAMETER in zppy v3.0.0:
+# scratch
+active = True
+walltime = "02:00:00"
+years = "1985:2014:30",
+
+[e3sm_diags]
+active = True
+multiprocessing = True
+num_workers = 8
+ref_final_yr = 2014
+ref_start_yr = 1985
+ts_num_years = 30
+walltime = "4:00:00"
+years = "1985:2014:30",
+
+  [[ atm_monthly_180x360_aave ]]
```

Comment:
I would prefer to have parameters related to obs path grouped together

Reply:
That would break the very intentional order of the variables in this cfg explaining which sets which require which extra parameters (this section breaks that down into 6 categories). My interpretation of this example cfg is to teach users what parameters are needed when. 

Grouping them in other ways may make sense for users, but I don't think it's necessarily the point of this particular cfg.

Reply:
^ @chengzhuzhang